app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+    if user&.authenticate params[:session][:password]
```

Comment:
Trả lời câu hỏi: toán tử &. nghĩa là sao?

Reply:
&. có nghĩa là vừa check nil vừa check condition đó if user&.authenticate params[:session][:password]